src/docs/product/data-management-settings/dynamic-sampling/index.mdx

```diff
@@ -3,33 +3,44 @@ title: Dynamic Sampling
 sidebar_order: 1
 redirect_from:
   - /product/data-management-settings/server-side-sampling/
-description: "Learn about how you can control which transactions Sentry retains using dynamic sampling."
+  - /product/data-management-settings/server-side-sampling/current-limitations/
+  - /product/data-management-settings/server-side-sampling/sampling-configurations/
+  - /product/data-management-settings/dynamic-sampling/current-limitations/
+  - /product/data-management-settings/dynamic-sampling/sampling-configurations/
+description: "Learn about Sentry dynamically prioritizes your transaction data."
 ---
 
-<Include name="limited-avail-note.mdx" />
+Capturing a single [trace](/product/sentry-basics/tracing/distributed-tracing/) — which consists of one or more transactions — involves minimal overhead, but capturing traces for every page load or every API request might overload your system. Sentry automatically prioritizes your transactions, server-side, so that we can provide you with the most valuable insights about your application’s performance while also applying helpful prioritization to the data we retain for you.
```

Comment:
This is a pitch for client-side sampling. With server-side sampling, every trace is still captured and sent to the server.

Reply:
Thanks for the heads up. I'll take that out!